ast.c

```diff
@@ -355,6 +356,7 @@ static inline zend_bool ast_kind_uses_attr(zend_ast_kind kind) {
 		|| kind == ZEND_AST_PROP_GROUP
 		|| kind == ZEND_AST_GROUP_USE || kind == ZEND_AST_USE_ELEM
 		|| kind == AST_NAME || kind == AST_CLOSURE_VAR || kind == ZEND_AST_CLASS_CONST_DECL
+        || kind == ZEND_AST_CLASS_CONST_GROUP
```

Comment:
There are some whitespace issues in the patch.

Reply:
Fixed, I didn't find any others